packages/itwinui-css/src/text/skeleton.scss

```diff
@@ -9,11 +9,11 @@
   cursor: progress;
   background: linear-gradient(
     292deg,
-    var(--iui-color-foreground-4),
-    var(--iui-color-foreground-4),
-    var(--iui-color-foreground-5),
-    var(--iui-color-foreground-4),
-    var(--iui-color-foreground-4)
+    var(--iui-color-text-muted),
```

Comment:
would `icon` colors look better here? these look too dark

i'm unsure if these need to meet the text contrast requirements or not

Reply:
I think your suggestion makes sense.  Text requires a higher score than icons because text is so thin, so using the icon variables and meeting 3.1 should be acceptable.  Change made.